contexts/chr15m/virtual-audio-graph.md

```diff
@@ -1,627 +1,213 @@
 ---
 title: virtual-audio-graph library
+provenance: distilled from the documentation
 ---
-## Virtual Audio Graph documentation
+# Virtual Audio Graph
 
-virtual-audio-graph is a library for declaratively manipulating the Web Audio API that is inspired by React and virtual-dom. This guide aims to introduce it step-by-step to help you get up and running as quickly as possible.
+A declarative Web Audio API library inspired by React's virtual DOM approach.
 
-### Browser support
+## Core Concepts
 
-virtual-audio-graph needs to be run in an environment that supports the web audio API (https://caniuse.com/#feat=audio-api). Make sure you check browser support for all the audio nodes you intend on using as newer audio nodes are often not supported in older browsers.
+- Create a virtual audio graph instance that manages Web Audio nodes
+- Declaratively specify audio node connections and parameters
+- Updates handle all imperative Web Audio API operations automatically
 
-## Importing
+## Basic Usage
 
-First let's import everything we're going to need for these examples:
-
-```js
+```javascript
 import createVirtualAudioGraph, {
-  bufferSource,
-  createNode,
-  createWorkletNode,
-  delay,
-  gain,
   oscillator,
-  stereoPanner,
+  gain,
+  OUTPUT
```

Comment:
@chr15m this seems like it might be wrong? 

Also, consider adding model and prompt to the provenance frontmatter. You can use markdown in that key. 

This way others can learn from how you distill documentation, which I think is a super interesting workflow anyways!

Reply:
> this seems like it might be wrong?

Can you provide me some more information. What do you think is wrong?

Ok, I'll add more info to provenance. I'm using [this script](https://gist.github.com/chr15m/1e52c9a246c2f8867325db3dd7085cd4) almost always so I'll add a note about that in future.

Reply:
Nevermind this seems correct! 👍